diff --git a/node_modules/amazon-cognito-identity-js/es/CookieStorage.js b/node_modules/amazon-cognito-identity-js/es/CookieStorage.js
index f3706fc..f61eb62 100644
--- a/node_modules/amazon-cognito-identity-js/es/CookieStorage.js
+++ b/node_modules/amazon-cognito-identity-js/es/CookieStorage.js
@@ -1,4 +1,4 @@
-import * as Cookies from 'js-cookie';
+import Cookies from 'js-cookie';
 
 /** @class */
 var CookieStorage = /*#__PURE__*/function () {
diff --git a/node_modules/amazon-cognito-identity-js/lib/CookieStorage.js b/node_modules/amazon-cognito-identity-js/lib/CookieStorage.js
index 6807df8..3ef4fa8 100644
--- a/node_modules/amazon-cognito-identity-js/lib/CookieStorage.js
+++ b/node_modules/amazon-cognito-identity-js/lib/CookieStorage.js
@@ -2,7 +2,7 @@
 
 exports.__esModule = true;
 exports["default"] = void 0;
-var Cookies = _interopRequireWildcard(require("js-cookie"));
+var Cookies = require("js-cookie").default;
 function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(e) { return e ? t : r; })(e); }
 function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { "default": e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n["default"] = e, t && t.set(e, n), n; }
 /** @class */
